### PR TITLE
pluma-view.c: use one static variable instead two to do the same job

### PR DESCRIPTION
**Pull request by sc0w**
This is the improvement of this two commits:

https://github.com/mate-desktop/pluma/commit/4df44887f4b7547fc9b31d1cb3f61688579fe39b
https://github.com/mate-desktop/pluma/commit/ec438f3d262ef178a898612211335d3c9fa2f4ee

The tests: make sure you can't reproduce the issues #250 and #256

### CODE REVIEW

File: pluma/pluma-view.c [hunk 1]
```diff
@@ -2034,13 +2032,17 @@ static gboolean
 pluma_view_button_release_event (GtkWidget *widget, GdkEventButton *event)
 {
 	if (event->button == 2)
-		middledown = FALSE;
-	else if (event->button == 3)
-		rightdown = FALSE;
+		middle_or_right_down = FALSE;
```
Comment (raveit65) [inline pluma/pluma-view.c]: Is is OK to remove everything for event->button == 3 here ?

Comment (sc0w) [reply pluma/pluma-view.c]: yes, never works the button 3 here (middle click), I did instead the function pluma_view_populate_popup

you can test that, it fixes a little issue:

1- select text

2- right button (appears the menu)

3- middle button (dissapears the menu)

4- middle button again

- the expected, with this PR: paste the text

- without this PR: nothing happens, it needs one more middle click

Comment (raveit65) [reply pluma/pluma-view.c]: Ok, got it

Comment (monsta) [reply pluma/pluma-view.c]: Confirmed the issue and the fix.

**Pull request merged**